admin/src/main/resources/io/buoyant/admin/js/router_summary.js

```diff
@@ -14,11 +14,12 @@ var RouterSummary = (function() {
   }
 
   function processResponse(data, routerName, metricName) {
-    var datum = Query.find(Query.clientQuery().withRouter(routerName).withMetric(metricName).build(), data);
-    return !datum ? 0 : datum.delta;
+    var datum = Query.filter(Query.serverQuery().allServers().withRouter(routerName).withMetric(metricName).build(), data);
+    return _.sumBy(datum, "delta");
   }
 
   function getSuccessAndFailureRate(result) {
+    if (_.isUndefined(result.failures)) result.failures = null;
```

Comment:
Oliver's branch (#258) adds `|| 0` to these arguments, so I think I'll remove this line